pytorch_to_returnn/torch/__init__.py

```diff
@@ -6,4 +6,5 @@
 from .autograd import *
 from .jit import *
 from .onnx import *
+from .cuda import *
```

Comment:
Why do you do this? Is this in the original Torch as well? I.e. there is a global function `torch.is_available`? That doesn't really make sense...

Reply:
That's true, doesn't make sense. I'll remove it.

Reply:
Isn't it the same for `jit`? Can we just remove it or should it instead be `import torch.cuda` and `import torch.jit`?

Reply:
Probably. I haven't checked it.
In any case, it should be just the same as in the original PyTorch.